task/validate-fbc/0.1/validate-fbc.yaml

```diff
@@ -450,27 +450,10 @@ spec:
         else
           echo -n "$related_images" > /shared/related-images.json
 
-          # A while loop executed in a subshell so we will check for failure based
-          # on whether /tmp/failedimages exists.
+          # We have moved verification of the related images to a Conformal policy. This enables
```

Comment:
s/Conformal/Conforma/ ?

Reply:
Yes, that was a mistake but I don't think it is enough to warrant a new task version.